packages/synthetics-sdk-broken-links/test/unit/link_utils.spec.ts

```diff
@@ -549,4 +553,40 @@ describe('GCM Synthetics Broken Links Utilies', async () => {
     ).to.be.lessThanOrEqual(new Date(syntheticResult.end_time).getTime());
     expect(syntheticResult.runtime_metadata).to.not.be.undefined;
   });
+
+  describe('shuffleAndTruncate', () => {
+    const links: LinkIntermediate[] = [
+      { target_url: 'link1', html_element: '', anchor_text: '' },
+      { target_url: 'link2', html_element: '', anchor_text: '' },
+      { target_url: 'link3', html_element: '', anchor_text: '' },
+      { target_url: 'link4', html_element: '', anchor_text: '' },
+      { target_url: 'link5', html_element: '', anchor_text: '' },
+    ];
+    const random =
+      BrokenLinksResultV1_BrokenLinkCheckerOptions_LinkOrder.RANDOM;
+    const firstN =
+      BrokenLinksResultV1_BrokenLinkCheckerOptions_LinkOrder.FIRST_N;
+    it('shuffles links when link_order is RANDOM', () => {
```

Comment:
a little bit of padding goes a long way, add a new line before each `it` in this file?